src/components/interaction-designer/toolbar/BlockErrorsExpandable.vue

```diff
@@ -83,11 +88,33 @@ export class BlockErrorsExpandable extends mixins(Lang) {
   }
 
   get resourceValidationStatusesForCurrentBlock(): IValidationStatus {
-    return get(this.validationStatuses, `resource/${this.block.config?.prompt}`)
+    const choiceResourceUuids = this.block.config?.choices !== undefined
+      ? Object.values(this.block.config.choices)
+      : []
+
+    return [
+      this.block.config?.prompt,
+      ...choiceResourceUuids,
+    ]
+      .map((uuid) => this.getAjvErrorsFor('resource', uuid))
+      .filter(Boolean)
   }
 
   get blockValidationStatusesForCurrentBlock(): IValidationStatus {
-    return get(this.validationStatuses, `block/${this.block.uuid}`)
+    return this.getAjvErrorsFor('block', this.block.uuid)
+  }
+
+  getAjvErrorsFor(type: 'block' | 'resource', uuid: string): null | Array<ErrorObject> {
+    const validationStatus = get(this.validationStatuses, `${type}/${uuid}`)
+
+    if (validationStatus === undefined) {
+      return null
+    }
+
+    return validationStatus.ajvErrors?.map((ajvError) => ({
+      ...ajvError,
+      dataPath: `${type}/${uuid}${ajvError.dataPath}`,
```

Comment:
can we add a separator `/` between `${uuid}` & `${ajvError.dataPath}`?

Reply:
The template looks inconsistent, but `dataPath` already contains a slash

Reply:
ah ok, thanks for confirming